src/Auth/AuthCookie.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Authentication Cookie.
+ *
+ * @package WPGraphQL\Login\Auth
+ * @since 0.0.1
```

Comment:
We'll update this on release
```suggestion
 * @since @todo
```